seq2science/rules/configuration_workflows.smk

```diff
@@ -184,6 +187,10 @@ def sieve_bam(configdict):
     )
 
 
+# after all is done, explain the workflow if requested
```

Comment:
I think just as with importing files it's better to leave the includes on the top.

Reply:
We're using includes in varying places already. However, this could be an opportunity to make the code a bit more pythonic. Ill make a 2nd PR with an idea...

Reply:
see #511